templates/gsl.configure.ac

```diff
@@ -847,6 +847,10 @@ AS_CASE([${$(my.symbol)}], [yes],
 .   define my.prefix = is_true(my.dependency.unprefixed) ?? "" ? "lib"
 .   define my.symbol = find_option_symbol(my.dependency, my.configure)?
 .   define my.check = "$(my.prefix)$(my.name) >= $(my.version)"
+.   if (defined(my.symbol))
+.       define my.option = my.configure->option(option.name =  my.dependency.option)
+.       define my.opttype = get_option_type(my.option)
```

Comment:
`my.opttype`

Style: avoid abbreviations and don't combine distinct words. For example, use `my.option.type`.